lib/line-protocol.js

```diff
@@ -73,10 +73,10 @@ internals.values = {
         {},
         internals.formatError(event.error),
         {
-            id     : internals.String(event.id),
-            url    : internals.String(event.url && Url.format(event.url)),
-            method : internals.String(event.method && event.method.toUpperCase()),
-            tags   : internals.String(event.tags)
+            id: internals.String(event.id),
```

Comment:
@fhemberger do you prefer the format with all colons lining up?  I don't have an opinion either way.